<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-BQC46TD3" ignored="0" name="Test" tests="5" failures="1" timestamp="2022-04-01T13:01:56 IST" time="0.125" errors="0">
  <testcase name="username1" time="0.016" classname="com.testng.feature.Demo_Runner_1"/>
  <testcase name="password" time="0.0" classname="com.testng.feature.Demo_Runner_1">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.0" classname="com.testng.feature.Demo_Runner_1">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.0" classname="com.testng.feature.Demo_Runner_1">
    <skipped/>
  </testcase> <!-- password -->
  <testcase name="password" time="0.0" classname="com.testng.feature.Demo_Runner_1">
    <failure type="java.lang.AssertionError" message="expected [Aravind] but found [Aravind@123]">
      <![CDATA[java.lang.AssertionError: expected [Aravind] but found [Aravind@123]
at com.testng.feature.Demo_Runner_1.password(Demo_Runner_1.java:26)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- password -->
</testsuite> <!-- Test -->
